src/coreclr/jit/regset.h

```diff
@@ -74,13 +74,8 @@ class RegSet
     bool rsModifiedRegsMaskInitialized; // Has rsModifiedRegsMask been initialized? Guards against illegal use.
 #endif                                  // DEBUG
 
-#ifdef SWIFT_SUPPORT
-    regMaskTP rsAllCalleeSavedMask;
-    regMaskTP rsIntCalleeSavedMask;
-#else  // !SWIFT_SUPPORT
-    static constexpr regMaskTP rsAllCalleeSavedMask = RBM_CALLEE_SAVED;
-    static constexpr regMaskTP rsIntCalleeSavedMask = RBM_INT_CALLEE_SAVED;
-#endif // !SWIFT_SUPPORT
+    regMaskTP rsAllCalleeSavedMask = RBM_CALLEE_SAVED;
```

Comment:
Does it mean that we are leaving a bit of perf on the table by turning into into a regular field?

Reply:
We will find out in the tpdiffs. If it does impact TP, I will revert it back.

Reply:
The underlying methods i.e. `rsClearRegsModified` and `rsGetModifiedCalleeSavedRegsMask` is just called once during the compilation, so I am guessing this shouldn't be much of an impact.

Reply:
No TP impact

![image](https://github.com/dotnet/runtime/assets/12488060/9ff86e8e-bd46-4e21-b79d-19bb36279797)